app/v2/hateoas/HateoasLinks.scala

```diff
@@ -122,11 +122,11 @@ trait HateoasLinks {
   }
 
   def createAmendHistoricFhlUkPropertyAnnualSubmission(appConfig: AppConfig, nino: String, taxYear: String): Link = {
-    Link(href = ukHistoricFhlAnnualUri(appConfig, nino, taxYear), method = PUT, rel = "create-and-amend-historic-fhl-uk-property-annual-submission")
+    Link(href = ukHistoricFhlAnnualUri(appConfig, nino, taxYear), method = PUT, rel = "create-and-amend-uk-property-historic-fhl-annual-submission")
   }
 
   def deleteHistoricFhlUkPropertyAnnualSubmission(appConfig: AppConfig, nino: String, taxYear: String): Link = {
-    Link(href = ukHistoricFhlAnnualUri(appConfig, nino, taxYear), method = DELETE, rel = "delete-historic-fhl-uk-property-annual-submission")
+    Link(href = ukHistoricFhlAnnualUri(appConfig, nino, taxYear), method = DELETE, rel = "delete-uk-property-historic-fhl-annual-submission")
```

Comment:
Can u add unit tests for these methods in HateoasLinksSpec, they are missing currently

Reply:
Added tests now